e2e-tests/constants/routes/your-business.js

```diff
@@ -0,0 +1,7 @@
+const INSURANCE_ROOT = '/insurance';
```

Comment:
Should these not be loaded from constants?

Reply:
This is in constants?

Reply:
Understood. thanks for the clarificaction.